src/python/WMCore/WMBS/Oracle/Locations/New.py

```diff
@@ -18,7 +18,7 @@ class New(NewLocationMySQL):
                                WHERE site_name = :location)"""
 
     seSQL = """INSERT INTO wmbs_location_senames (location, se_name)
-                 SELECT wl.id, :se_name FROM wmbs_location wl
+                 SELECT wl.id, :pnn FROM wmbs_location wl
                  WHERE wl.site_name = :location
                  AND NOT EXISTS (SELECT null FROM wmbs_location_senames wls2 WHERE
                                   wls2.se_name = :se_name
```

Comment:
Missed changing that se_name to pnn.

With this it now at least deploys, lets see if it works...